libraries/transformer/src/androidTest/java/androidx/media3/transformer/mh/HdrEditingTest.java

```diff
@@ -360,6 +397,74 @@ public void onFallbackApplied(
     }
   }
 
+  @Test
+  public void exportAndTranscode_dolbyVisionFile_whenHdrEditingUnsupported_toneMapsOrThrows()
+      throws Exception {
+    Context context = ApplicationProvider.getApplicationContext();
+    Format format = MP4_ASSET_DOLBY_VISION_HDR_FORMAT;
+    assert format.colorInfo != null;
+    if (deviceSupportsHdrEditing(VIDEO_H265, format.colorInfo)) {
+      recordTestSkipped(context, testId, /* reason= */ "Device supports Dolby Vision editing.");
+      return;
```

Comment:
We can possibly use assumeDeviceDoesNotSupportHdrEditing() ?

Reply:
Sorry, I'm not familiar with resolving confliction on GitHub. Let's complete other issues then handle the confliction issue. (I fork the main branch at the beginning of this code contribution. At that time, there is no this method.)